### PR TITLE
metrics: Adds blogbench and webtool metrics tests

### PR DESCRIPTION
**Pull request by dborquez**
This PR adds blogbench and webtooling metrics checks to this repo.

Fixes: #7069

### CODE REVIEW

File: tests/metrics/storage/blogbench_dockerfile/Dockerfile.in
Comment (GabyCT) [inline tests/metrics/storage/blogbench_dockerfile/Dockerfile.in:1]: @dborquez I do not see that this file is being used

Comment (dborquez) [reply tests/metrics/storage/blogbench_dockerfile/Dockerfile.in:1]: good catch, so let me remove it

**Comment by amshinde** [comment]
@dborquez lgtm. Just wanted to clarify with you about how the tests would be run. These would need to be run on a dedicated runner in order to get reliable storage numbers. Running this on a VM provided by GHA would give us unreliable numbers.

**Comment by dborquez** [comment]
@amshinde that's correct, we have a dedicated BM runner w/ label 'metrics'.

**Review by GabyCT — APPROVED** [review]

File: tests/metrics/gha-run.sh [hunk 1]
```diff
@@ -91,12 +91,22 @@ function run_test_launchtimes() {
...
+}
+
 function main() {
 	action="${1:-}"
 	case "${action}" in
 		install-kata) install_kata ;;
 		run-test-launchtimes-qemu) run_test_launchtimes "qemu" ;;
 		run-test-launchtimes-clh) run_test_launchtimes "clh" ;;
+		run-test-blogbench-qemu) run_test_blogbench "qemu" ;;
+		run-test-blogbench-clh) run_test_blogbench "clh" ;;
```
Comment (fidencio) [inline tests/metrics/gha-run.sh:109]: I'm missing something here, but shouldn't we also add those to the netrics yaml action?

I'd take the following path:
* Add to the yaml
* Add it here, with short-circuit to always return true

In a separe PR:
* Actually enable the tests in the script and make sure they're passing.

Comment (amshinde) [reply tests/metrics/gha-run.sh:109]: @dborquez You will need to modify `run-k8s-tests-on-aks.yaml` in order to invoke these tests.

File: tests/metrics/storage/webtooling.sh [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+set -o pipefail
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+NUM_CONTAINERS="$1"
+
+TEST_NAME="web-tooling"
+IMAGE="docker.io/library/local-web-tooling:latest"
+DOCKERFILE="${SCRIPT_PATH}/web-tooling-dockerfile/Dockerfile"
```
[the remark this answers is not in the document]
Comment (dborquez) [reply tests/metrics/storage/webtooling.sh:20]: @GabyCT it was discarded, thanks

File: tests/metrics/gha-run.sh (hunk 1 above)
Comment (dborquez) [reply tests/metrics/gha-run.sh]: @fidencio I've make the function that runs the test in order to returns zero
I've added the test to the workflow, but I will wait that the memory test to land, and rebase this PR w/ the 'matrix' usage  update.

Comment (dborquez) [reply tests/metrics/gha-run.sh]: @amshinde I took the matrix idea from there, thank you :)

**Review by fidencio — APPROVED** [review]
lgtm, thanks @dborquez!

**Pull request merged**